addon/run-task.js

```diff
@@ -153,22 +162,27 @@ export function scheduleTask(obj, queueName, taskOrName, ...args) {
    @param { Number } [timeout] the time in the future to run the task
    @public
    */
-export function throttleTask(obj, name, timeout = 0) {
-  assert(
-    `Called \`throttleTask\` without a string as the first argument on ${obj}.`,
-    typeof name === 'string'
-  );
-  assert(
-    `Called \`throttleTask('${name}', ${timeout})\` where '${name}' is not a function.`,
-    typeof obj[name] === 'function'
-  );
+export function throttleTask(obj, taskOrName, timeout = 0) {
```

Comment:
I'd prefer to keep these changes separate, as they're not really related to this PR per se. Is it OK if we move them out?